staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -80,6 +81,21 @@ func createHandler(r rest.NamedCreater, scope RequestScope, admit admission.Inte
 			return
 		}
 
+		options := &metav1.CreateOptions{}
+		values := req.URL.Query()
+		// TODO: replace with content type negotiation?
+		// "1" is accepted as "true" for backward compatibility.
+		for i, v := range values["includeUninitialized"] {
```

Comment:
this could be moved into a helper maybe. Also, do we regression test this?

Reply:
Convert_Slice_string_To_bool should already handle this conversion